packages/vela/src/utils.ts

```diff
@@ -72,6 +72,35 @@ export function getOrderTypePacked(
   }
 }
 
+export function getOrderType(
+  orderType: OrderType | undefined,
+): { _orderType: undefined } | { $or: { _orderType: number }[] } | undefined {
+  if (!orderType) return { _orderType: undefined }
+  const orderTypeValues = {
+    [OrderType.Market]: {
+      $or: [{ _orderType: 0 }, { _orderType: 2 }],
+    },
+
+    [OrderType.Limit]: {
+      $or: [{ _orderType: 1 }, { _orderType: 3 }],
+    },
+  }
+  return orderTypeValues[orderType]
+}
+
+export function getAmount(amount: Amount): FilterOperator | undefined {
```

Comment:
Would be good to document why this is necessary in relation to their protocol and how they handle value with some function comments.